### ISSUE TITLE
wav2vec: Get an exception error saying minimum loss scale (0.0001) reached 

### ISSUE THREAD

**Issue by shamanez** [pytorch/fairseq#968]
I used the training script provided in the wav2vec repository to train libraspeech 960-hour dataset. The only difference is, I have 2 1080 GPUs. I didn't change the --update-freq parameters. So the effective batch size is 16, not 64. I got this error after 17 epochs. What is the reason behind this?
